Functions/Quantification/xASL_quant_CBF.m

```diff
@@ -292,18 +298,48 @@
 fprintf('%s\n',' model with parameters:');
 
 if x.modules.asl.ApplyQuantification(3)
```

Comment:
Seems to make code easier if you initialize PLDs & LDs before this part as `[]`, and give a warning instead when you cannot find them. Also, is LD an agreed upon abbreviation? Otherwise, `LabDur` could be clearer.

Reply:
OK. Sure, question of style, but I have nothing against. Will do.